### ISSUE TITLE
💡 [FEATURE] - Adding subtle scroll animations. 

### ISSUE THREAD

**Issue by yashi-15** [jfmartinz/ResourceHub#719]
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

We can enhance this website by adding scroll animations throughout. This will display the content with a subtle animation effect as users scroll through the site.

Kindly assign this issue to me under GSSOC'24 so that I can start working on it.

**Comment by jfmartinz** [comment]
Scroll to top feeature?

**Comment by yashi-15** [comment]
> Scroll to top feeature?

No, @jfmartinz, the content on the webpage will fade in as the user scrolls through the site. 
I'll be using the ScrollReveal library from JavaScript to animate elements as they come into view.

**Comment by jfmartinz** [comment]
Okay, sure. Use seperate js adn make it descriptive
